### DISCUSSION TITLE
Administração de Caravana

### DISCUSSION THREAD

**Discussion by JhordhanCarlos** [diego3g/rsxp-2023#7] [Ideas]
O diego falou na live que vai rolar umas caravanas de diversos estados para o evento, seria bacana adicionar uma feature para quem organiza essas caravanas, para aqueles que já estão com os ingressos dentro o app e para aqueles que desejam ir para o evento mas ainda não tem certeza.

**Comment by viniseven** [comment]
Se na autenticação do usuário, o usuário inserir sua região, e inserir seu número de telefone, daria por dentro do app, buscar pessoas que irão participar no evento, filtrando por regiões e a partir daí, poder com uma funcionalidade abrir o whatsapp para falar com essa pessoa e combinar saída do ponto de origem juntos. etc

**Reply by ruymon** [reply]
O app poderia solicitar a localização e, a partir desse dado, sugerir algumas caravanas que estão abertas ainda.

**Reply by LuisCarlosDev** [reply]
isso seria muito massa!

**Reply by LuisCarlosDev** [reply]
Mais isso seria só para as pessoas que se inscreveram no evento não é isso?

**Reply by JhordhanCarlos** [reply]
> Mais isso seria só para as pessoas que se inscreveram no evento não é isso?

Acho que não, pois no começo da live o diego falou que o intuito do MVP do App seria divulgar o evento.

**Reply by viniseven** [reply]
@LuisCarlosDev essa funcionalidade em específico sim, como vão usar a api no sympla, daria pra deixar aessa funcionalidade para quem tem ingresso adquirido.

**Reply by brunoesposito** [reply]
@viniseven mas ter isso disponível para quem nao comprou ainda seria uma coragem a mais se o cara tiver sozinho. Ai cobre um pouco do que o @JhordhanCarlos que já que vai ser bem divulgado, a conversão de compra poderia ser maior.

**Reply by viniseven** [reply]
@brunoesposito nesse caso deixaria aberto pra todos. apenas exibindo um icon ou alguma info no usuário que tem ingresso adquirido.

**Reply by trolliama** [reply]
Essa funcionalidade seria muito interessante para pessoas que moram longe como eu!! Poderia até mesmo ter uma página simples mostrando caravanas que já estão certas de que irão acontecer perto da sua região. Desse modo ao invés da pessoa entrar em contato com outra sem saber se a caravana vai seguir em frente um perfil poderia cadastrar uma caravana no app (nesse caso as infos seriam apenas local de partida, data, hora e o numero do organizador) e outros usuários entrariam em contato.

**Comment by GbCarvalho** [comment]
Ae @JhordhanCarlos, muito daora tua sugestão, acho que ela acaba se enquadrando em alguns dos tópicos que o Diego levantou na live, a gente não está se preocupando com o que vai acontecer no evento, mas sim antes do evento nesse momento e acaba que estamos incentivando a ida ao evento

**Reply by JhordhanCarlos** [reply]
Sim, acaba difundindo o próprio evento para mais pessoas. Não sei se apenas quem já comprou o ingresso vai ter acesso ao App, mas se não tiver a pessoa pode ver a caravana da cidade(ou estado) e comprar o ingresso pelo App(através do Sympla) e já ter a experiência completa.

**Comment by diego3g** [comment]
Galera, ideia incrível! Vou validar internamente com o time se existe timing pra fazermos isso via app ou se é algo que pela necessidade de tempo precisa rodar fora do app nesse momento, ok?

**Comment by diego3g** [comment]
Galera, to pensando em facilitarmos nessa primeira versão. Penso em no app termos links para grupos no WhatsApp ou Discord sobre cada caravana ao invés de gerenciarmos tudo dentro do app como quem está dentro de cada caravana, etc, porque acho que pode ser um pouco demais para o momento atual. O que acham?

**Reply by rafael-hc** [reply]
Poderia ter links de grupos por região assim acho que já ajudaria bastante as pessoas se encontrarem. Cada usuário entra no grupo da sua região.

**Reply by JhordhanCarlos** [reply]
É, pra primeira versão seria mais complexo, com ideia do @rafael-hc pode ser mais simples. Se bobear a parte de gerenciamento de caravana pode até ser um App a parte(só mandar a UI e as regras de negócio que a gente desembola kkkkk)

**Reply by Rodrigo001-dev** [reply]
Será que não seria legal listar links de grupos baseado na localização do dispositivo do usuário?

**Reply by trolliama** [reply]
De fato apenas uma página que filtra as caravanas de acordo com região já seria o suficiente para isso, tendo links de whatsapp, telegram, discord, etc. Além dessas informações achho válido também ter a informação do local de partida para que o usuário já no app tenha uma noção se ele terá condições de ir até determinado local

**Reply by carlosdavid0** [reply]
@Rodrigo001-dev  ideia bacana. Porém, isso ia partir contra a simplicidade que o Diego sugeriu.

**Reply by trolliama** [reply]
@carlosdavid0 Mas acho que é algo que vai melhorar e muito a experoência de usuário. De qualquer forma será preciso de pelo menos um campo para que o usuário filtre por região (não apenas um local fixo), pois muitas caravanas podem sair de uma cidade ao lado do que a que o usuário mora, então essas deveriam ser mostradas também

**Reply by carlosdavid0** [reply]
@trolliama Concordo muito.

**Reply by JhordhanCarlos** [reply]
@trolliama Pra deixar mais simples, as vezes, o usuário poderia só habilitar a localização é já aparecer uma lista com as caravanas em volta do raio da cidade dele.

**Reply by diego3g** [reply]
Galera, tudo muito massa, mas lembrem que simplicidade no início é fundamental e, aos poucos, podemos ir melhorando a feature! Bora sempre pensar na versão mais simples que entrega valor disso.

Por exemplo, será que uma página no Notion com cada caravana e cada região e um link pra isso dentro do app não resolveria? :)
